build/Common.prod.props

```
@@ -59,6 +59,19 @@
     <Message Importance="high" Text="**AssemblyVersionDebug** TargetFramework: $(TargetFramework), MinVerVersion: $(MinVerVersion), MinVerMajor: $(MinVerMajor), MinVerMinor: $(MinVerMinor), MinVerPatch: $(MinVerPatch), MinVerPreRelease: $(MinVerPreRelease), BuildNumber: $(BuildNumber), FileVersion: $(FileVersion), ExposeExperimentalFeatures: $(ExposeExperimentalFeatures)" />
   </Target>
 
+  <Target Name="ResolveExposeExperimentalFeatures" AfterTargets="MinVer">
```

Shouldn't this be run after `AssemblyVersionTarget`?

```suggestion
  <Target Name="ResolveExposeExperimentalFeatures" AfterTargets="AssemblyVersionTarget">
```

---

I had that at first but `AssemblyVersionTarget` gets skipped for non-publish builds and we want this section to always run.

---

When publishing builds, is the ordering guaranteed? We don't want this target to be evaluated before `AssemblyVersionTarget`.

---

Not 100% sure about that, but I changed it so it now has a `DependsOnTargets="AssemblyVersionTarget"`